---
title: "LIONESS"
date: 2019-06-02T08:50:52-04:00
draft: false
---

{{< rawhtml >}}
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
{{< /rawhtml >}}
{{< rawhtml >}}
<div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.isci.2019.03.021" data-hide-no-mentions="true" class="altmetric-embed"></div>
{{< /rawhtml >}}
### Reference

Estimating Sample-Specific Regulatory Networks. 

Marieke Lydia Kuijjer, Matthew George Tung, Guo Cheng Yuan, John Quackenbush, Kimberly Glass

iScience

[doi.org/10.1016/j.isci.2019.03.021](https://www.sciencedirect.com/science/article/pii/S2589004219300872).

### Abstract

Biological systems are driven by intricate interactions among molecules. Many methods have been developed that draw on large numbers of expression samples to infer connections between genes (or their products). The result is an aggregate network representing a single estimate for the likelihood of each interaction, or “edge,” in the network. Although informative, aggregate models fail to capture population heterogeneity. Here we propose a method to reverse engineer sample-specific networks from aggregate networks. We demonstrate our approach in several contexts, including simulated, yeast microarray, and human lymphoblastoid cell line RNA sequencing data. We use these sample-specific networks to study changes in network topology across time and to characterize shifts in gene regulation that were not apparent in the expression data. We believe that generating sample-specific networks will greatly facilitate the application of network methods to large, complex, and heterogeneous multi-omic datasets, supporting the emerging field of precision network medicine.

### Implementation

[netZooR](https://github.com/netZoo/netZooR), [netZooPy](https://github.com/netZoo/netZooPy), [netZooM](https://github.com/netZoo/netZooM)

### Netbook tutorials

The following [netbooks](http://netbooks.networkmedicine.org) use LIONESS:

- netZooR:

	- Building PANDA and LIONESS Regulatory Networks from GTEx Gene Expression Data in R

	- Comparing LIONESS Regulatory Networks using limma

	- Differential gene targeting of pancreatic cancer subtypes

	- Building and analyzing enhancer-driven TF-gene regulatory networks PANDA and LIONESS
